.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -80,17 +80,6 @@
                 ]),
             ]),
         ]),
-        ("gcc", [
-            ("9", [
-                ("3.8", [
-                    ("coverage", [
-                        (True, [
-                            ("shard_test", [XImportant(True)]),
-                        ]),
-                    ]),
-                ]),
-            ]),
-        ]),
```

Comment:
These lines were manually removed & then `regenerate.sh` was run to create `config.yml`.